pkg/metricscollector/prommetrics.go

```diff
@@ -35,16 +36,16 @@ var (
 		prometheus.GaugeOpts{
 			Namespace: DefaultPromMetricsNamespace,
 			Name:      "build_info",
-			Help:      "A metric with a constant '1' value labeled by version, git_commit and goversion from which KEDA was built.",
+			Help:      "Info metric, with static information about KEDA build like: version, git commit and Golang runtime info.",
```

Comment:
nit: was the comma intentional?
```suggestion
			Help:      "Info metric with static information about KEDA build like: version, git commit and Golang runtime info.",
```